pkg/formatter/formatters.go

```diff
@@ -121,3 +121,29 @@ func (f simpleJsonFormatter) Write(out io.Writer, set querier.SeriesSet) error {
 
 	return err
 }
+
+type nilFormatter struct {
+	baseFormatter
+}
+
+func (f nilFormatter) Write(out io.Writer, set querier.SeriesSet) error {
+	var count int
+	for set.Next() {
+		count++
+		series := set.At()
+		iter := series.Iterator()
+		for iter.Next() {
```

Comment:
what's the purpose of this empty loop?

Reply:
wanted to exhaust the iterator, but isnt really needed ill delete it.